dietpi/func/dietpi-logclear

```diff
@@ -117,13 +118,13 @@
 					((INFO_BACKUPS_MADE++))
 
 					# Clear log file contents
-					> "$i"
+					2> /dev/null > "$i" || ARRAY_LOGFILE_TRUNCATE_DENIED+=("$i")
```

Comment:
I think the STDERR redirect like that does nothing? Did you want to mute possible errors messages of the STDOUT redirect? I think that would only work like this:
```sh
{ > "$i"; } 2> /dev/null
```
But as I mentioned in the issue, I am no fans of redirection STDERR, as long as we do not always expect errors. There might be important errors, like hardware I/O issues, filesystem corruption and whatnot, what one should see. So I prefer to completely switch to `truncate`:
```suggestion
					ARRAY_LOGFILE_TRUNCATE_DENIED+=("$i")
```
Could be named differently then, of course.

Reply:
`2> /dev/null > "$i"` actually does work in muting stderr (tested). `> "$i" 2> /dev/null` does not work.